src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -782,20 +803,25 @@ void ApplicationManagerImpl::OnHMIStartedCooperation() {
   rpc_service_->ManageHMICommand(mixing_audio_supported_request);
   resume_controller().ResetLaunchTime();
 
-  CollectCloudAppInformation();
+  RefreshCloudAppInformation();
 }
 
-void ApplicationManagerImpl::CollectCloudAppInformation() {
-  std::vector<std::string> cloud_app_id_vector;
-  GetPolicyHandler().GetEnabledCloudApps(cloud_app_id_vector);
-  std::vector<std::string>::iterator it = cloud_app_id_vector.begin();
-  std::vector<std::string>::iterator end = cloud_app_id_vector.end();
+void ApplicationManagerImpl::RefreshCloudAppInformation() {
```

Comment:
This functions behavior and purpose is hard to understand at first. Function could use some more comments describing the flow.

Reply:
Added comments